### DISCUSSION TITLE
Agenda for the Flatcar Developer Sync on Wednesday, 26th of June 2024, 2:30pm UTC

### DISCUSSION THREAD

**Discussion by t-lo** [flatcar/Flatcar#1464] [Flatcar Developer Sync]
In our monthly Flatcar developer syncs maintainers review development / roadmap progress, discuss special projects like ClusterAPI and sysext, provide an overview of ongoing work, and give an outlook of future work.

Join if you're curious about current or future Flatcar development and/or to raise visibility of a feature or bug.

- Jitsi Link: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording:  https://www.youtube.com/live/pLZZPvbXUkI
  - All recordings: https://www.youtube.com/@FlatcarLinux/live
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Jitsi call. Please introduce yourself and share what brings you here today.

## News
- Docker 25 is coming (@t-lo )

## Monthly Contributions and Advisory status Review
1. Discuss the latest [Monthly Contributions Report](https://github.com/flatcar/Flatcar/issues?q=is%3Aissue+is%3Aopen+Monthly+contributions+report+label%3Akind%2Fmetric)
   * First-Time Contributors in last month:
     * @nachtjasmin [website](https://github.com/flatcar/flatcar-website/commits?author=nachtjasmin&since=2024-05-22&until=2024-06-21)
     * @alejandro-ripoll [bootengine](https://github.com/flatcar/bootengine/commits?author=alejandro-ripoll&since=2024-05-22&until=2024-06-21)
2. Review of last month's advisory stats in the latest [Contributions Report](https://github.com/flatcar/Flatcar/issues?q=is%3Aissue+is%3Aopen+Monthly+contributions+report+label%3Akind%2Fmetric)
   a. Advisory status and planning: https://github.com/orgs/flatcar/projects/7/views/17

## Ongoing and Planned Tasks Review

1. Roadmap review: https://github.com/orgs/flatcar/projects/7/views/9 (@t-lo, @miao0miao )
2. Projects status review:
   a. Discuss retiring / merging CAPI and Sysext efforts into "composable OS"
   b. Discuss introducing new project "SecureBoot"
   c. CNCF Incubation process (@miao0miao); https://github.com/orgs/flatcar/projects/7/views/18
   d. New vendors / platforms developments (@tormath1)
4. Maintainer team tactical / implementation (aka "line work") status review (@sayanchowdhury): https://github.com/orgs/flatcar/projects/7/views/1
5. Open Mic: Work to assign / User issues and PRs to discuss?

### Release Planning
- Ongoing and upcoming releases tracker: (https://github.com/orgs/flatcar/projects/7/views/8) (@sayanchowdhury)
- Talk about the upcoming release and the status of the packages
- Status of the ongoing release
- Preparation of the next release 

### Q&A
- Questions from community participants, answered by the Flatcar maintainers 

### Administrative tasks after meeting concluded
- [x] Create a new Github discussion for the next Flatcar developer sync: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-developer-sync
- [x] Close this discussion as "Resolved".- [ ]

**Comment by tormath1** [comment]
Similar to last time I can give a brief status overview on the new Flatcar images / platforms.

**Reply by t-lo** [reply]
That's actually a good point, I'll take it into our agenda.
